src/azure-cli/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -238,7 +238,13 @@
 helps['keyvault create'] = """
 type: command
 short-summary: Create a Vault or HSM.
-long-summary: If `--enable-rbac-authorization` is not specified, then default permissions are created for the current user or service principal unless the `--no-self-perms` flag is specified.
+long-summary: >-
+    Default permissions are created for the current user or service principal unless the `--no-self-perms` 
+    or `--enable-rbac-authorization` flag is specified. 
```

Comment:
This changes the original cumbersome expression (https://github.com/Azure/azure-cli/pull/12074#discussion_r1360540526).